generate_table.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+"""Script for generation table with data/models from list.json in .md format"""
+import argparse
+import json
+import os
+
+
+DATASETS_SOURCE = "list.json"
+
+
+def generate_table(fn):
+    with open(fn) as infile:
+        data = json.loads(infile.read())
+
+    datasets = sorted(data["corpora"].items(), key=lambda kv: kv[0])
+    models = sorted(data["models"].items(), key=lambda kv: kv[0])
+
+    print("## Available data")
+    print("### Datasets")
+    print("| name | source | description |")
+    print("|------|--------|-------------|")
+    for name, other in datasets:
+        if name.startswith("__testing_"):
+            continue
+        print("| {name} | {source} | {description} |".format(
+            name=name, source=other["source"], description=other["description"]
+        ))
+
+    print("")
+    print("### Models")
+    print("| name | description | papers | preprocessing | parameters |")
+    print("|------|-------------|------------|--------|---------------|")
+    for name, other in models:
+        if name.startswith("__testing_"):
+            continue
+        print("| {name} | {description} | {papers} | {preprocessing} | {parameters} |".format(
+            name=name, description=other["description"], parameters=other.get("parameters", "-"),
+            papers=other.get("papers", "-"), preprocessing=other.get("preprocessing", "-")
+        ))
+    print("\n(generated by {} based on {})".format(__file__, os.path.basename(fn)))
```

Comment:
No need to hide the path -- very useful for debugging and clarity (multiple Python installations by mistake etc).